components/account/AccountMoreButton.vue

```diff
@@ -62,6 +68,22 @@ const toggleReblogs = async () => {
 
       <template v-if="currentUser">
         <template v-if="!isSelf">
+          <template v-if="relationship!.following">
```

Comment:
I'm getting an error when testing here, the relationship could be `undefined`
```suggestion
          <template v-if="relationship?.following">
```

Reply:
@patak-dev The menu item was removed here, so the code is gone now.